stardog/requirements.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: zookeeper
   repository: https://charts.bitnami.com
-  version: 2.2.4
-digest: sha256:a732f9cdf32e878df52f619d8adde164bb8d73ba3457c85c9fd558509865d1a8
-generated: "2019-06-24T12:59:27.128477698+02:00"
+  version: 5.0.1
```

Comment:
Does every node have a snapshot file in the data directory? https://github.com/bitnami/charts/tree/master/bitnami/zookeeper/#to-300